src/EditorFeatures/TestUtilities/Diagnostics/AbstractUserDiagnosticTest.cs

```diff
@@ -195,10 +195,6 @@ protected Document GetDocumentAndAnnotatedSpan(TestWorkspace workspace, out stri
 
             var actions = fixes.SelectAsArray(f => f.Action);
 
-            actions = actions.SelectMany(a => a is TopLevelSuppressionCodeAction
-                ? a.NestedCodeActions
-                : ImmutableArray.Create(a)).ToImmutableArray();
```

Comment:
why removed?

Reply:
The logic was specific to suppression actions, so has been moved into the MassageActions override in AbstractSuppressionDiagnosticTest